### PR TITLE
HDDS-6340. [Multi-Tenant] Add tenant CLI option to print results in JSON

### PR DESCRIPTION
**Pull request by smengcl**
## What changes were proposed in this pull request?

For tenant read commands, add CLI option (`--json`) to print results in JSON.

For tenant write commands, use `--verbose` parameter to optionally print JSON result if applicable.

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-6340

## How was this patch tested?

- [x] Update unit test
- [x] Update existing documentation

### CODE REVIEW

**Comment by smengcl** [comment]
@aswinshakil Would you take a look at this? Thx!

**Review by aswinshakil — COMMENTED** [review]
Overall LGTM. Few minor comments inline. Thank you for the patch!

File: hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/shell/tenant/TenantDeleteHandler.java [hunk 1]
```diff
@@ -35,30 +38,36 @@ public class TenantDeleteHandler extends TenantHandler {
...
-            + "Ozone features (refCount is " + volumeRefCount + ").");
-      }
-    } catch (IOException e) {
-      // Throw exception to make client exit code non-zero
-      throw new IOException("Failed to delete tenant '" + tenantId + "'", e);
+
+    final DeleteTenantState resp =
+        client.getObjectStore().deleteTenant(tenantId);
+
+    err().println("Deleted tenant '" + tenantId + "'.");
```
[the remark this answers is not in the document]
Comment (smengcl) [reply hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/shell/tenant/TenantDeleteHandler.java:48]: Yeah this was debated earlier with @errose28 last week.

The point is to have a nice prompt that informs the admin to delete the volume manually.

stderr doesn't necessarily have to only contain error message though (e.g. try `wget google.com 2> /dev/null`). The point here is to seperate the JSON output (printed to stdout) and user prompts.

**Review by aswinshakil — APPROVED** [review]
+1 LGTM. Thank you for the patch @smengcl.

**Comment by smengcl** [comment]
Thanks @aswinshakil for reviewing this.

**Pull request merged**